[external] bugfix: fix escaping in orders

bobbyshaw opened the pull request:
# [DEVDOCS-]
The OpenAPI parser was unable to comprehend the escaping approach of this example on an orders endpoint. I've modified it to use single quotes so that the double quotes don't need to be escaped. 

## What changed?
Provide a bulleted list in the present tense
* change escaping of orders example for improved ability to parse.

## Anything else?
Add related PRs, salient notes, ticket numbers, etc.

ping {names}

slsriehl commented:
@bobbyshaw can you resolve conflicts to incorporate https://github.com/bigcommerce/api-specs/commit/5c27dd44bed3d9d0313768fabd3b0a9a190e7664?

In addition, can you provide a response to document that the strings received do NOT include the `\` escape literals? I'm trying to determine which is more accurate -- `'{"foo": "bar"}'` or `'{\"foo\": \"bar\"}'`.

bobbyshaw commented:
Hi Sarah, I've rebased. 

Thanks for your suggestion to check the response document. I found that the response document does have the escapes:

```
[
    {
        "id": 11,
        "order_id": 107,
        "product_options": [
            {
                "id": 9,
                "option_id": 37,
                "order_product_id": 11,
                "product_option_id": 116,
                "display_name": "Logo",
                "display_name_customer": "Logo",
                "display_name_merchant": "Logo",
                "display_value": "BigCommerce-logo-dark.png",
                "display_value_customer": "BigCommerce-logo-dark.png",
                "display_value_merchant": "BigCommerce-logo-dark.png",
                "value": "{\"originalName\":\"BigCommerce-logo-dark.png\",\"temporaryPath\":\"116_db8ccbbddf55c73bc7d6bcc2ae10b459.png\",\"path\":\"e214d766ffcf9476a592f341f4631f8a.png\"}",
                "type": "File upload field",
                "name": "Logo1694339766-112",
                "display_style": ""
            }
        ],
    }
]
```

I've updated my fix to just double-quote which still resolves my issue without changing the value example.
